src/Components/Table/Documentation/Table.stories.tsx

```diff
@@ -0,0 +1,368 @@
+// Libraries
+import React, {RefObject, createRef} from 'react'
+import marked from 'marked'
+
+// Storybook
+import {storiesOf} from '@storybook/react'
+import {withKnobs, select, text, boolean, object} from '@storybook/addon-knobs'
+import {mapEnumKeys} from '../../../Utils/storybook'
+import {jsxDecorator} from 'storybook-addon-jsx'
+
+// Components
+import {
+  Table,
+  TableRef,
+  TableHeaderRef,
+  TableHeaderCellRef,
+  TableBodyRef,
+  TableRowRef,
+  TableCellRef,
+  TableFooterRef,
+} from '../'
+
+// Types
+import {
+  Alignment,
+  ComponentSize,
+  BorderType,
+  ComponentColor,
+} from '../../../Types'
+
+// Notes
+import TableReadme from './Table.md'
+
+const tableStories = storiesOf('Components|Table/Family', module)
+  .addDecorator(withKnobs)
+  .addDecorator(jsxDecorator)
+
+tableStories.add(
+  'Table',
+  () => {
+    const tableRef: RefObject<TableRef> = createRef()
+    const tableHeaderRef: RefObject<TableHeaderRef> = createRef()
+    const tableHeaderCellRef: RefObject<TableHeaderCellRef> = createRef()
+    const tableBodyRef: RefObject<TableBodyRef> = createRef()
+    const tableRowRef: RefObject<TableRowRef> = createRef()
+    const tableCellRef: RefObject<TableCellRef> = createRef()
+    const tableFooterRef: RefObject<TableFooterRef> = createRef()
+
+    const logRefs = (): void => {
```

Comment:
Maybe we should make a utility for this since we're using it in all the stories.